packages/common/src/document-types.js

```diff
@@ -221,6 +221,14 @@ const documentTypes = {
 		involvement: '',
 		horizonDocumentType: 'Appellant Final Comments',
 		horizonDocumentGroupType: 'Evidence'
+	},
+	dynamic: {
```

Comment:
What's the use case for the dynamic document type?

Reply:
So the document api requires meta data to be added alongside the file upload which comes from these objects.

It also runs integration style tests against each of the document types from this file to ensure they can be uploaded and fetched again.

So a large part of this is so I didn't need to rewrite too much of the existing document upload logic/tests/potentially break the existing appeals journey

I was under the impression we would want to continue using this, but would be adding new document types for each upload question, at which point we have a single source of every document type we have.

However I wasn't sure what the meta data needed for the various uploads was going to be so I created a generic one as a placeholder (the tickets for uploads don't specify any metadata needed at the moment)

So:
Maybe we don't need and we need to refactor the existing document upload/tests
Maybe we can just leave as this as is, as a catch all for docs that don't need this metadata
Maybe we add a new doc type for each file upload question